src/style.css

```diff
@@ -0,0 +1,38 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars .stars__star {
+  border-radius: 0.5px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 .stars__star {
+  border-radius: 0.5px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
This code (besides` background-image: url(./images/star.svg);`) is fully copied. Try not to repeat yourself.

You can set these rules for `.stars__star` and only `background-image: url(./images/star.svg);` for `.stars--0 .stars__star`